source/User_Guide/Settings/Whitelabel/How_to_set_up_link_branding.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+weight: 0
+title: How to set up link branding
+seo:
+  title: How to set up link branding
+  description: Email link branding allows all of the click-tracked links and opens tracked images in your emails to be from your domain instead of from sendgrid.net.
+  keywords: whitelabeling, domain whitelabeling, link branding
+navigation:
+  show: false
+---
+
+- [What is link branding?]()
+- [Key terminology]()
+    - [Term]()
+    - [Term]()
+- [Setting up link branding]()
+    - [Before you begin]()
+- [Troubleshooting]()
+- [Migrating from legacy Whitelabel]()
+
+{% anchor h2 %}
+What is link branding?
+{% endanchor %}
+
+Email link branding allows all of the click-tracked links and opens tracked images in your emails to be from your domain instead of from sendgrid.net. Spam filters and recipient servers look at the links within emails to determine whether the email looks trustworthy enough to deliver - they use the reputation of the root domain to determine whether the links can be trusted. Implementing link labeling helps in email deliverability because you are no longer relying on click tracking going through a domain that you do not control.
+
+{% anchor h2 %}
+Key terminology
+{% endanchor %}
+
+{% anchor h3 %}
+DNS
+{% endanchor %}
+
+{% anchor h3 %}
+Term
+{% endanchor %}
+
+{% anchor h2 %}
+Setting up link branding
+{% endanchor %}
+
+{% anchor h3 %}
+Before you begin
+{% endanchor %}
+
+To set up link branding, you must submit the DNS records provided by SendGrid to your DNS or hosting provider (for example, GoDaddy, Hover, CloudFlare, etc.). First, you need to figure out who your hosting provider is and if you have access. If you don't have access to your DNS or hosting provider, you should figure out who in your company has this access before you begin setting up link branding.
+
+{% info %}
+Note that you can also set up link branding when you set up your domain authentication and you may not need to set it up again.
+{% endinfo %}
+
+*To set up and verify link branding:*
+
+1. In the SendGrid UI, select [Settings > Sender Authentication](https://app.sendgrid.com/settings/whitelabel).
+1. In the link branding section, click **Get Started**.
```

Comment:
How does it look in the UI? Is it "Link Branding" or "link branding"?

Reply:
I don't actually know. UI comes out this afternoon and they never changed the name in the designs